src/chapters/classes-and-objects/why-use-objects.rst

```diff
@@ -181,16 +181,4 @@ Check Your Understanding
 
 .. Answer = c
 
-.. raw:: html
-
-   <script type="text/JavaScript">
-      function evaluateMC(id, correct) {
-         if (correct) {
-            document.getElementById(id).innerHTML = 'Yep!';
-            document.getElementById(id).style.color = 'blue';
-         } else {
-            document.getElementById(id).innerHTML = "Nope! This refers to the object's methods.";
-            document.getElementById(id).style.color = 'red';
-         }
-      }
-   </script>
```

Comment:
This script element might need to stay here.  It's got a question-specific feedback message.

Reply:
added back in to rst file